psl/psl/tests/validation/relations/set_null_is_not_valid_on_required_fields.prisma

```diff
@@ -0,0 +1,26 @@
+model SomeUser {
+  id      Int      @id
+  profile Profile?
+}
+
+model Profile {
+  id      Int       @id
+  user    SomeUser? @relation(fields: [user_id], references: [id], onUpdate: SetNull, onDelete: SetNull)
+  user_id Int       @unique
+}
```

Comment:
This schema with one required referenced field in a  relation with the `SetNull` referential action was considered valid by Prisma. After this PR, it's not.

Reply:
:+1: